android-app/app/build.gradle

```diff
@@ -25,7 +25,7 @@ android {
         targetSdk 33
         versionCode 1
         versionName "1.0"
-
+        buildConfigField("String", "TIME_ZONE", "\"Europe/Berlin\"")
```

Comment:
Is it correct to escape the string quotation marks here like this `"\"Europe/Berlin\""` instead of entering it directly like this `"Europe/Berlin"`?

Reply:
https://stackoverflow.com/questions/30796533/how-to-generate-buildconfigfield-with-string-type
I think that should be right